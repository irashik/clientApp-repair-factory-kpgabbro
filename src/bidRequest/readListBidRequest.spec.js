


import React from 'react';
import { shallow, render } from 'enzyme';

import ReadListBidRequest from './readListBidRequest';



describe('<ReadListBidRequest />', () => {

    test('snapShot correctly', () => {

        const component = shallow(<ReadListBidRequest />);
        expect(component).toMatchSnapshot();

    });
});


