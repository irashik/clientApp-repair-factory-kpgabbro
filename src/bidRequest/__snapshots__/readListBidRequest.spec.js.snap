// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReadListBidRequest /> snapShot correctly 1`] = `
<div>
  Нет заявок
</div>
`;
